ansible/provision.yaml

```diff
@@ -29,7 +29,6 @@
     - include_vars: vars/flatcar/flatcar.yaml
       when: ansible_os_family == "Flatcar"
   roles:
-    - role: offline
```

Comment:
We should actually keep this here. In the case where the user is trying to build an AMI with the offline bundles present